### PR TITLE
[ISSUE #4088] Anonymous new can be replaced with lambda.[BroadCastSubClient]

### PR DESCRIPTION
**Pull request by AXE-00**
Issue Fix: #4088 
Replaced the anonymous inner class with a lambda expression for the handle method

### CODE REVIEW

File: eventmesh-runtime/src/test/java/org/apache/eventmesh/runtime/demo/BroadCastSubClient.java [hunk 1]
```diff
@@ -40,15 +40,11 @@ public static void main(String[] args) throws Exception {
...
-                    if (msg.getHeader().getCommand() == Command.BROADCAST_MESSAGE_TO_CLIENT) {
-                        if (msg.getBody() instanceof EventMeshMessage) {
-                            String body = ((EventMeshMessage) msg.getBody()).getBody();
-                            LogUtils.info(log, "receive message -------------------------------{}", body);
-                        }
+            client.registerBusiHandler((msg, ctx) -> {
+                if (msg.getHeader().getCommand() == Command.BROADCAST_MESSAGE_TO_CLIENT) {
+                    if (msg.getBody() instanceof EventMeshMessage) {
+                        String body = ((EventMeshMessage) msg.getBody()).getBody();
+                        LogUtils.info(log, "receive message -------------------------------{}", body);
```
[the remark this answers is not in the document]
Comment (AXE-00) [reply eventmesh-runtime/src/test/java/org/apache/eventmesh/runtime/demo/BroadCastSubClient.java]: Modified the suggested changes. Do I have to create another PR?

File: eventmesh-runtime/src/test/java/org/apache/eventmesh/runtime/demo/BroadCastSubClient.java [hunk 2]
```diff
@@ -44,10 +42,12 @@ public static void main(String[] args) throws Exception {
                 if (msg.getHeader().getCommand() == Command.BROADCAST_MESSAGE_TO_CLIENT) {
                     if (msg.getBody() instanceof EventMeshMessage) {
                         String body = ((EventMeshMessage) msg.getBody()).getBody();
-                        LogUtils.info(log, "receive message -------------------------------{}", body);
+                        if (log.isInfoEnabled()) {
+                            log.info("receive message : {}", body);
+                        }
```
Comment (Pil0tXia) [inline eventmesh-runtime/src/test/java/org/apache/eventmesh/runtime/demo/BroadCastSubClient.java:47]: By changing " -------------------------------" to ": ", there should not be any additional space between the word "message" and the colon ":".

Besides, why `LogUtils` was substituted with logger?

File: eventmesh-runtime/src/test/java/org/apache/eventmesh/runtime/demo/BroadCastSubClient.java [hunk 3]
```diff
@@ -44,10 +42,12 @@ public static void main(String[] args) throws Exception {
...
                     }
                 }
             });
         }
-    }
+    }
```
Comment (Pil0tXia) [inline eventmesh-runtime/src/test/java/org/apache/eventmesh/runtime/demo/BroadCastSubClient.java:52]: Please mind Checkstyle.

**Review by Pil0tXia — CHANGES_REQUESTED** [review]

File: eventmesh-runtime/src/test/java/org/apache/eventmesh/runtime/demo/BroadCastSubClient.java (hunk 3 above)
Comment (AXE-00) [reply eventmesh-runtime/src/test/java/org/apache/eventmesh/runtime/demo/BroadCastSubClient.java]: Done!

Comment (AXE-00) [reply eventmesh-runtime/src/test/java/org/apache/eventmesh/runtime/demo/BroadCastSubClient.java]: Fixed codestyle. Sorry, I'm quite new to this, making a lot of mistakes at every step. Thanks for guiding

**Review by mxsm — APPROVED** [review]
LGTM

**Pull request merged**